TransformationSystem/ConfigTemplate.cfg

```diff
@@ -32,6 +32,34 @@ Agents
   }
   TransformationCleaningAgent
   {
+    # MetaData key to use to identify output data
+    TransfIDMeta=TransformationID
+
+    # Location of the OutputData, if the OutputDirectories parameter is not set for
+    # transformations only 'MetadataCatalog has to be used
+    DirectoryLocations=TransformationDB,MetadataCatalog
+
+    # From which SEs files will be removed.
+    # If empty or not existing, only files in the Catalog will be removed
+    ActiveSEs=
+
+    # Enable or disable, default enabled
+    EnableFlag=True
+
+    # How many days to wait before archiving transformations
+    ArchiveAfter=7
+
+    # Shifter to use for removal operations, default is empty and
+    # using the transformation owner for cleanup
+    shifterProxy=
+
+    # Which transformation types to clean
+    # If not filled, transformation types are taken from
+    #   Operations/Transformations/DataManipulation
+    # and Operations/Transformations/DataProcessing
+    TransformationTypes=
+
+    #Time between cycles in seconds
     PollingTime = 120
```

Comment:
Should this be updated to something more reasonable 24h?

Reply:
24h seems a lot. Maybe 1h?